pkg/ddl/executor.go

```diff
@@ -6778,6 +6780,12 @@ func (e *executor) DoDDLJobWrapper(ctx sessionctx.Context, jobW *JobWrapper) (re
 		// Instead, we merge all the jobs into one pending job.
 		return appendToSubJobs(mci, jobW)
 	}
+	// all DDLs should set InvolvingSchemaInfo to make sure DDL execution in order.
+	for _, info := range job.GetInvolvingSchemaInfo() {
+		if info.Database == model.InvolvingNone || info.Table == model.InvolvingNone {
+			return dbterror.ErrInvalidDDLJob.GenWithStack("invalid ddl job args: InvolvingSchemaInfo")
+		}
+	}
```

Comment:
please wrap it like `checkHistoryJobInTest`, only run when `intest.EnableInternalCheck` is `true`, this check is mostly to avoid human error when we add new DDL